ui/src/pages/publications/[id]/index.tsx

```diff
@@ -200,19 +206,32 @@ const Publication: Types.NextPage<Props> = (props): React.ReactElement => {
     );
 
     const onBookmarkHandler = async () => {
-        if (isBookmarked) {
+        if (showBookmark) {
```

Comment:
```suggestion
        if (isBookmarked) {
```

We can replace with 'isBookmarked'